src/main/java/com/puppycrawl/tools/checkstyle/meta/JavadocMetadataScraper.java

```diff
@@ -51,6 +51,12 @@
 @FileStatefulCheck
 public class JavadocMetadataScraper extends AbstractJavadocCheck {
 
+    /**
+     * A key is pointing to the warning message text in "messages.properties"
+     * file.
+     */
+    public static final String DESC_MISSING = "javadoc.description.missing";
```

Comment:
Please rename to `MSG_DESC_MISSING` to follow our other patterns.

Reply:
done